webpack/webpack-plugin-utilities/src/index.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+/**
+ * Utility package which provides a set of tools for woking in
+ * webpack plugins, loaders, and other integrations.
+ * @packageDocumentation
+ */
+
+export { isWebpack3OrEarlier, isWebpack4Or5 } from './DetectWebpackVersion';
```

Comment:
```suggestion
export { VersionDetection } from './DetectWebpackVersion';
```

Reply:
And then rename the `DetectWebpackVersion.ts` file too.

Reply:
Or 
```suggestion
export * as VersionDetection from './DetectWebpackVersion';
```

Reply:
The latter works for me

Reply:
I’ll get to this Monday and rebase @iclanton’s changes from his PR